.github/ISSUE_TEMPLATE/technical-issue-report-by-contributor.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Technical Issue report by contributor
+about: Describe a discovered problem with the code in more technical detail
+title: "[Detailed Issue]"
+labels: ''
+assignees: ''
+
+---
+
+**Discovered problem**
```

Comment:
imo, we can merge this one with bug reports as one template since it is more like a package